frontend/src/components/CostTableMenu.vue

```diff
@@ -14,43 +14,73 @@
     </n-checkbox-group>
   </div>
 
-  <div class="mt-4 space-x-2 flex justify-start">
+  <div class="mt-2 flex flex-wrap">
+    <!-- Min specification for Memory & vCPU -->
+    <div class="inline-block align-bottom max-w-min mr-2 pt-2">
+      <n-input-number
```

Comment:
this control doesn't feel right to input the number directly, there is no input cursor. My feeling is it's more used for changing the amount (like a ticker used in the shopping cart).

I would suggest to use a normal text input only accepting integers or just use a dropdown with some predefined values (e.g 1, 2, 4, 8 and etc)

Reply:
I slightly change the style, making the +/- button disappear and showing user the input cursor.
wdyt?